src/service_cmd/runner/runner.go

```diff
@@ -75,6 +77,16 @@ func createLimiter(srv server.Server, s settings.Settings, localCache *freecache
 
 func (runner *Runner) Run() {
 	s := runner.settings
+	if s.TracingEnabled {
```

Comment:
I want to make sure I understand how this is implemented: Right now all of these traces are independent, right? Because we aren't passing any incoming span data, right? Are you still planning on changing the API to optionally pass on context?

Reply:
Resolving this conversation as it is a duplicate.